wicket-jquery-ui-samples/pom.xml

```diff
@@ -15,7 +15,7 @@
 
 	<properties>
 		<jetty.version>9.4.44.v20210927</jetty.version>
-		<slf4j.version>2.0.0-alpha2</slf4j.version>
+		<slf4j.version>2.0.0</slf4j.version>
```

Comment:
Better use the latest - https://central.sonatype.com/artifact/org.slf4j/slf4j-api/2.0.9

Reply:
updated